server/odc-service/src/main/java/com/oceanbase/odc/service/regulation/risklevel/RiskLevelService.java

```diff
@@ -96,6 +96,13 @@ public RiskLevel findDefaultRiskLevel() {
                         .orElseThrow(() -> new NotFoundException(ResourceType.ODC_RISK_LEVEL, "level", 0)));
     }
 
+    @SkipAuthorize("internal usage")
+    public RiskLevel findHighestRiskLevel() {
+        return entityToModel(
+                riskLevelRepository.findByOrganizationIdAndLevel(authenticationFacade.currentOrganizationId(), 3)
```

Comment:
the highest level is 2

Reply:
NoNoNo, it is 3. RiskLevel: default(0), low(1), medium(2), high(3).